spec/models/steps/parallel_spec.rb

```diff
@@ -1,58 +1,60 @@
-describe Steps::Parallel do
-  let(:proposal) { create(:proposal) }
-
-  it "allows approvals in any order" do
-    first = build(:approval_step, proposal: nil)
-    second = build(:approval_step, proposal: nil)
-    third = build(:approval_step, proposal: nil)
-    proposal.root_step = Steps::Parallel.new(child_steps: [first, second, third])
-
-    expect(proposal.root_step.reload.status).to eq("actionable")
-    expect(first.reload.status).to eq("actionable")
-    expect(second.reload.status).to eq("actionable")
-    expect(third.reload.status).to eq("actionable")
-
-    first.complete!
-    expect(proposal.root_step.reload.status).to eq("actionable")
-    expect(first.reload.status).to eq("completed")
-    expect(second.reload.status).to eq("actionable")
-    expect(third.reload.status).to eq("actionable")
-
-    third.complete!
-    expect(proposal.root_step.reload.status).to eq("actionable")
-    expect(first.reload.status).to eq("completed")
-    expect(second.reload.status).to eq("actionable")
-    expect(third.reload.status).to eq("completed")
-
-    second.complete!
-    expect(proposal.root_step.reload.status).to eq("completed")
-    expect(first.reload.status).to eq("completed")
-    expect(second.reload.status).to eq("completed")
-    expect(third.reload.status).to eq("completed")
-  end
-
-  it "can be used for disjunctions (ORs)" do
-    first = build(:approval_step, proposal: nil)
-    second = build(:approval_step, proposal: nil)
-    third = build(:approval_step, proposal: nil)
-    proposal.root_step = Steps::Parallel.new(min_children_needed: 2, child_steps: [first, second, third])
-
-    first.reload.complete!
-    expect(proposal.root_step.reload.status).to eq("actionable")
-    expect(first.reload.status).to eq("completed")
-    expect(second.reload.status).to eq("actionable")
-    expect(third.reload.status).to eq("actionable")
-
-    third.complete!
-    expect(proposal.root_step.reload.status).to eq("completed")
-    expect(first.reload.status).to eq("completed")
-    expect(second.reload.status).to eq("actionable")
-    expect(third.reload.status).to eq("completed")
-
-    second.complete!
-    expect(proposal.root_step.reload.status).to eq("completed")
-    expect(first.reload.status).to eq("completed")
-    expect(second.reload.status).to eq("completed")
-    expect(third.reload.status).to eq("completed")
-  end
-end
+# Not using Parallel Steps.
```

Comment:
Yeah... but then i'd like to delete the code too, and that's a much bigger change. I commented this out because these are the slowest tests. So this is an optimization, not necessarily a clean-up.